contrib/checkbox-ce-oem/checkbox-provider-ce-oem/units/video-codec/jobs.pxu

```diff
@@ -27,3 +27,30 @@ flags: also-after-suspend
 environ: GST_LAUNCH_BIN
 command:
     gst_v4l2_video_decoder_md5_checksum_comparison.py -dp {{decoder_plugin}} -cs {{color_space}} -gp {{golden_sample_file}} -gmp {{golden_md5_checkum_file}}
+
+unit: template
+template-engine: jinja2
+template-resource: video_codec_resource
+template-filter: video_codec_resource.scenario == "gst_v4l2_audio_video_synchronization"
+template-unit: job
+template-id: ce-oem-video-codec/gst_v4l2_audio_video_synchronization
+_template-summary: To check if the relative timing of audio and video is synchronized under a specific decoder
+id: ce-oem-video-codec/{{ scenario }}-{{ decoder_plugin }}-{{ golden_sample_file_name }}-{{ video_sink }}
+_summary: AV-Sync test of decoder {{ decoder_plugin }} with {{ golden_sample_file_name }} file
+_purpose: To check if the relative timing of audio and video of {{ golden_sample_file_name }} file is synchronized under a specific {{ decoder_plugin }} decoder
+plugin: user-interact-verify
+category_id: video-codec
+estimated_duration: 30s
+imports: from com.canonical.plainbox import manifest
+requires: manifest.has_video_codec == 'True'
+flags: also-after-suspend
+environ: GST_LAUNCH_BIN
+_steps:
+    1. Plug a headset/earphone to the device
+    2. Adjust the audio sink to be headset via Gnome Settings or command
+    3. Press enter to perform test
+_verification:
+    Do you hear the beep sound and see the video be displayed?
+    Is the timing of audio and video synchronized?
```

Comment:
Ensure there's no delay between the audio output and the visual display.

Reply:
![Screenshot from 2024-07-29 09-56-27](https://github.com/user-attachments/assets/7554d919-c53f-4ab8-9608-01e2d2ff5533)

@stanley31huang How about this version?

Reply:
![Screenshot from 2024-07-29 10-06-25](https://github.com/user-attachments/assets/a0d57410-b7f7-4e6c-b10d-17783dcaf8ca)